plugins/commando.c

```diff
@@ -804,6 +804,10 @@ static struct command_result *reply_with_rune(struct command *cmd,
 
 	json_add_string(js, "rune", rune_to_base64(tmpctx, rune));
 	json_add_string(js, "unique_id", rune->unique_id);
+
+	if (tal_count(rune->restrs) <= 1) {
+		json_add_string(js, "warning_usage", "WARNING: This rune has superadmin permissions. Anyone who has access to this rune could drain funds from your node. Be careful when giving this to apps that you don't trust. Consider using the restrictions parameter to only allow access to specific rpc methods.");
```

Comment:
"This rune has no restrictions!" ... is clearer (and more precise IMHO!).

Reply:
:+1: done